msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
+
+**Adaptive Card responsive layout with `targetWidth` configuration**
+
+:::image type="content" source="../../assets/images/Cards/adaptive-card-responsive-layout.png" alt-text="Screenshot shows how adaptive card renders when the targetWidth is configured to respective card widths.":::
+
+#### Sample Adaptive Card responsive layout
+
+To make a responsive Adaptive Card, your app must include the elements from the following code sample:
+
+``` json
+{ 
+  "type": "AdaptiveCard", 
+  "body": [ 
+    { 
+      "type": "ColumnSet", 
+      "columns": [ 
+        { 
+          "type": "Column", 
+          "items": [ 
+            { 
+              "type": "Image", 
+              "style": "Person", 
+              "url": "https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg", 
+              "size": "Small"
+            } 
+          ], 
+          "width": "auto" 
+        }, 
+        { 
+          "type": "Column", 
+          "spacing": "medium", 
+          "items": [ 
+            { 
+              "type": "TextBlock", 
+              "weight": "Bolder", 
+              "text": "Matt Hidinger", 
+              "wrap": true 
+            }, 
+            { 
+              "type": "TextBlock", 
+              "targetWidth": "narrow", 
+              "spacing": "None", 
+              "text": "Principal Program Manager", 
+              "isSubtle": true, 
+              "wrap": true 
+            } 
+          ], 
+          "width": "auto", 
+          "verticalContentAlignment": "center" 
+        }, 
+        { 
+          "type": "Column", 
+          "targetWidth": "atLeast:standard", 
+          "spacing": "medium", 
+          "items": [ 
+            { 
+              "type": "TextBlock", 
+              "text": "Principal Program Manager", 
+              "isSubtle": true, 
+              "wrap": true 
+            } 
+          ], 
+          "width": "stretch", 
+          "verticalContentAlignment": "center" 
+        } 
+      ] 
+    } 
+  ], 
+  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json", 
+  "version": "1.5" 
+} 
+```
+
+#### Responsive Adaptive Card layout design guidelines
+
+##### `targetWidth` set to `wide`
+
+You can use the following design guidelines for the `targetWidth` property set to `wide`:
+
+:::image type="content" source="../../assets/images/Cards/design-layout-wide.png" alt-text="Screenshot shows the design guidelines for a responsive adaptive card when you set target width to wide.":::
+
+|Design elements  |Description  |
+|---------|---------|
+| extraLarge spacing |  |
+| default spacing |  |
+
+###### Wide
+
+:::image type="content" source="../../assets/images/Cards/responsive-ac-wide.png" alt-text="Screenshot shows the responsive adaptive card when you set target width to wide.":::
+
+##### `targetWidth` set to `narrow`
```

Comment:
I do not understand how any of the below is going to be useful to developers. Either the whole below section needs to be redone or it should be removed IMO.